Content.Tests/DMProject/Broken Tests/Expression/String/multiline_whitespace.dm

```diff
@@ -0,0 +1,8 @@
+//# issue 23
+
+var/str = "A\
+		
+	B"
+
+/proc/RunTest()
+	ASSERT(str == "AB")
```

Comment:
This one's a little odd. This should work, but `str` appears to be null for some reason. If the declaration of `str` is moved to inside the proc then it works correctly.

Not really necessary to fix this now, though.